### DISCUSSION TITLE
Is breeding not a mechanic in this mod?

### DISCUSSION THREAD

**Discussion by purejosh** [MrKacafirekCZ/doggo-mod-overhauled#41] [Q&A] [answered]
I've tried every combination I can think of to try and breed my dogs, but it simply isn't working. Is breeding broken when you have this mod installed?

**Comment by MrKacafirekCZ** [comment, marked as answer]
Hello, ~~yes breeding is currently a bug/non-existent feature. You can find more about it in #25 .~~ For versions x.3.0 and up this bug has been fixed!
